frontend/src/metabase/query_builder/components/filters/modals/InlineCategoryPicker/InlineCategoryPicker.unit.spec.tsx

```diff
@@ -0,0 +1,337 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-nocheck
+import React from "react";
+import { render, screen, waitFor } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+import { metadata } from "__support__/sample_database_fixture";
+
+import Field from "metabase-lib/lib/metadata/Field";
+import Filter from "metabase-lib/lib/queries/structured/Filter";
+import Question from "metabase-lib/lib/Question";
+
+import { InlineCategoryPickerComponent } from "./InlineCategoryPicker";
+import { MAX_INLINE_CATEGORIES } from "./constants";
+
+const smallCategoryField = new Field({
+  database_type: "test",
+  semantic_type: "type/Category",
+  table_id: 8,
+  name: "small_category_field",
+  has_field_values: "list",
+  values: ["Michaelangelo", "Donatello", "Raphael", "Leonardo"],
```

Comment:
Do you know if any API would return values in this non-nested array form? Otherwise, would it be better to just use the value exactly like what the API would normally return?

e.g. Product category
```json
{
  "values": [["Doohickey"], ["Gadget"], ["Gizmo"], ["Widget"]],
  "field_id": 4,
}
```